### PR TITLE
Fixed styles on Wallet modal and added wallet type on NavbarProfileDropdown

### LINKED ISSUE #4302: Add name of wallet you are logged in and show assets of address
We should probably add the name of the account so users know under which name they are logged in.

@exezbcz will drop some ideas:)
image

### PR DESCRIPTION
**Pull request by fabiogatti**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [X] Bugfix
- [X] Feature
- [ ] Refactoring

## Context

- [X] Closes #4302
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality
- [X] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [X] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DdFn4hDQDTkzoH9dFPnsuE2VvmnHAhAn2RezVedZHVYaCEb) (I think the issue number doesnt have a bounty on it)

#### Community participation

- [X] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [X] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

The wallet searcher modal had a small space related bug, when the text was big it displaced the wallet to another space:
walletTextBug
Fixed:
walletTextBugFixed
(Also centered the "Connect Wallet" title in all of the modals and added some hover effects to the buttons)

Finally, it didnt have the wallet type that was connected:
walletNoType
Fixed:
walletWithType

### CODE REVIEW

File: components/common/WalletModal.vue [hunk 1]
```diff
@@ -291,8 +300,21 @@ export default class WalletModal extends mixins(UseApiMixin, ChainMixin) {
...
+      display: flex;
 
+      .header-btn {
+        padding: 10px !important;
+        border-radius: 4px;
+        position: absolute;
+      }
+      .delete {
+        right: 14px;
+      }
```
Comment (Jarsen136) [inline components/common/WalletModal.vue:313]: I see a duplicate class name `.delete` here.

**Review by Jarsen136 — COMMENTED** [review]
WFM

File: components/common/WalletModal.vue (hunk 1 above)
Comment (fabiogatti) [reply components/common/WalletModal.vue:313]: You are quite right, pushing the fix RN

**Comment by yangwao** [comment]
> users know under which name they are logged in.

Hey and can you post screenshot where I can see name of wallet name connected?

**Comment by fabiogatti** [comment]
> Hey and can you post screenshot where I can see name of wallet name connected?

Like this?

walletWithType

**Comment by yangwao** [comment]
oh I see now.

> add the name of the account so users know under which name they are logged in.

I mean like which name of account you are using, so in your case "My Polkadot Account" would be there somewhere, 
not wallet type/name :) 

I also see we miss design there sort of 😏

**Comment by fabiogatti** [comment]
Ah I get it now... should I wait for the design to drop or should I try to fix it as best as I can?
Also... should I then remove the icon and wallet type that is connected?

**Comment by exezbcz** [comment]
> @yangwao Ah I get it now... should I wait for the design to drop or should I try to fix it as best as I can? Also... should I then remove the icon and wallet type that is connected?

i will have it in 1 hour

**Comment by yangwao** [comment]
> should I wait for the design to drop

I guess the designs have arrived, let's continue :)

https://github.com/kodadot/nft-gallery/issues/4302#issuecomment-1332603449

**Comment by yangwao** [comment]
Hey @fabiogatti let me know if you want to work on this, otherwise, we would like to see it in our codebase soon as possible :)

**Comment by yangwao** [comment]
>  I think the issue number doesnt have a bounty on it

@fabiogatti and yes meanwhile there is bounty for this one!

**Pull request closed without merging**